app.js

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env node
 // shebang interferes with license header plugin
```

Comment:
having deleted the shebang we can remove the commant and disable I think?

Reply:
ah right